runtime/common/src/apis.rs

```diff
@@ -481,16 +481,17 @@ macro_rules! impl_runtime_apis_plus_common {
 						// pallet ethereum index: 1
 						// transact call index: 1
 						// Transaction enum variant: 1
-						// chain_id 8 bytes
+						// chain_id 9 bytes (some varient)
 						// nonce: 32
-						// max_priority_fee_per_gas: 32
-						// max_fee_per_gas: 32
+						// max_priority_fee_per_gas: 33 (some varient)
+						// max_fee_per_gas: 33 (some varient)
```

Comment:
Same here, `max_priority_fee_per_gas` and `max_fee_per_gas` have the same type as `nonce`. `U256`, which internally has the following structure `pub struct U256(pub [u64; 4]);` (1 byte + 4 * u64) = 33 bytes)

Reply:
Never mind, this is wrong.

Just tested this locally with `U256::default().encoded_size()` and it only uses 32 bytes.

Reply:
We should add a unit test that ensures we are using the correct size for these fields.

Reply:
Same as before it's wrapped in Option, so we need 1 extra byte